rm/rm-api/src/main/java/org/ow2/proactive/resourcemanager/frontend/RMMonitoring.java

```diff
@@ -42,7 +43,7 @@
  * @since ProActive Scheduling 0.9
  */
 @PublicAPI
-public interface RMMonitoring {
+public interface RMMonitoring extends Serializable {
```

Comment:
Is it a firm strict constraint from `PAFuture.getFutureValue` method ?

Reply:
In gneral an object returned by an active object method must be Serializable, but this interface exception is not really important here, I will probably remove it.